--- production
+++ candidate
@@ -7,8 +7,9 @@
 argv[3]=--validators-proposer-default-fee-recipient=0x1111111111111111111111111111111111111111
 argv[4]=--builder-endpoint=http://mevboost.my.ava.do:18550
 argv[5]=--validators-builder-registration-default-enabled=true
-argv[6]=--p2p-subscribe-all-subnets-enabled=true
-argv[7]=--log-destination=CONSOLE
+argv[6]=--Xp2p-quic-enabled=false
+argv[7]=--p2p-subscribe-all-subnets-enabled=true
+argv[8]=--log-destination=CONSOLE
 jwt secret file /data/data-gnosis/jwttoken: 66 bytes
 --- config file /data/config.yml
 # Generated file.
@@ -31,9 +32,12 @@
 # Execution engine
 ee-endpoint: "http://nethermind-gnosis.my.ava.do:8551"
 
-# metrics
-metrics-enabled: false # TODO: beaconchain monitor
-metrics-categories: ["BEACON","LIBP2P","NETWORK"]
+# metrics (Prometheus, scraped by prometheus.avado.dappnode.eth on the AVADO network only)
+metrics-enabled: true
+metrics-interface: "0.0.0.0"
+metrics-host-allowlist: ["*"]
+metrics-port: 8008
+metrics-categories: ["JVM","PROCESS","BEACON","DISCOVERY","EVENTBUS","EXECUTOR","LIBP2P","NETWORK","STORAGE","VALIDATOR","VALIDATOR_PERFORMANCE"]
 # Options are: JVM, PROCESS, BEACON, DISCOVERY, EVENTBUS, EXECUTOR, LIBP2P, NETWORK, STORAGE, STORAGE_HOT_DB, STORAGE_FINALIZED_DB, REMOTE_VALIDATOR, VALIDATOR, VALIDATOR_PERFORMANCE
 
 # database
